Build failed on Version 1.1.0

Once I update the version to **1.1.0** it gave build failure  but the plugin is working with version **1.0.0** 

```
> Task :generateSwagger FAILED
####################
Yelp Swagger Codegen
####################
Platform         kotlin
Package          com.yelp.codegen.integrations
specName         integration
specVers         1.0.0
input            /Users/chatura/swagger-gen/swagger-gradle-codegen-master/samples/sample_specs.json
output           /Users/chatura/swagger-gen/swagger-gradle-codegen-master/samples/groovy-android/src/main/java
groupId          com.yelp.codegen.integrations
artifactId       com.yelp.codegen.integrations
features         Accept-Language

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':generateSwagger'.
> com/yelp/codegen/utils/InlineModelResolver has been compiled by a more recent version of the Java Runtime (class file version 55.0), this version of the Java Runtime only recognizes class file versions up to 52.0
```


These are my configurations 


```
------------------------------------------------------------
Gradle 5.5
------------------------------------------------------------

Build time:   2019-06-28 17:36:05 UTC
Revision:     83820928f3ada1a3a1dbd9a6c0d47eb3f199378f

Kotlin:       1.3.31
Groovy:       2.5.4
Ant:          Apache Ant(TM) version 1.9.14 compiled on March 12 2019
JVM:          1.8.0_212 (AdoptOpenJDK 25.212-b04)
OS:           Mac OS X 10.13.6 x86_64
```

--- a/README.md
+++ b/README.md
@@ -10,7 +10,6 @@ This plugin wraps [swagger-codegen](https://github.com/swagger-api/swagger-codeg
 
 **Swagger Gradle Codegen** is distributed through [Gradle Plugin Portal](https://plugins.gradle.org/). To use it you need to add the following dependency to your gradle files.
 Please note that those code needs to be added the gradle file of the module where you want to generate the code (**not the top level** build.gradle\[.kts\] file).
-Please also note that the plugin uses gradle 5.+ features, and you will need to setup your gradle wrapper to use 5.0 or more.
 
 If you're using the Groovy Gradle files:
 ```groovy
@@ -52,6 +51,13 @@ generateSwagger {
 Please note that the `generateSwagger { }` block is **needed** in order to let the plugin work.
 
 Once you setup the plugin correctly, you can call the `:generateSwagger` gradle task, that will run the code generation with the configuration you provided.
+
+## Requirements
+
+In order to run this gradle plugin you need to fulfill the following requirements:
+
+* Gradle 5.x - This plugin uses Gradle 5 features, and you will need to setup your Gradle wrapper to use 5.0 or more.
+* Java 8+
 
 ## Supported platforms
 

--- a/plugin/build.gradle.kts
+++ b/plugin/build.gradle.kts
@@ -11,6 +11,11 @@ plugins {
     id("com.gradle.plugin-publish") version "0.10.0"
 }
 
+java {
+    sourceCompatibility = JavaVersion.VERSION_1_8
+    targetCompatibility = JavaVersion.VERSION_1_8
+}
+
 dependencies {
     implementation(kotlin("stdlib-jdk8"))
 

